pkg/services/vmoperator/vmopmachine_test.go

```diff
@@ -37,22 +37,22 @@ import (
 	"sigs.k8s.io/cluster-api-provider-vsphere/pkg/util"
 )
 
-func getReconciledVM(ctx *vmware.SupervisorMachineContext) *vmoprv1.VirtualMachine {
+func getReconciledVM(vmService VmopMachineService, supervisorMachineContext *vmware.SupervisorMachineContext) *vmoprv1.VirtualMachine {
```

Comment:
I think we should also pass in `ctx` into this function and not use the global var inside the helper function. WDYT?


```suggestion
func getReconciledVM(ctx context.Context, vmService VmopMachineService, supervisorMachineContext *vmware.SupervisorMachineContext) *vmoprv1.VirtualMachine {
```

(same for `updateReconciledVM`)

Reply:
Sounds good.